plugin.go

```diff
@@ -167,6 +172,26 @@ func (p *Plugin) vaultPass() error {
 	return nil
 }
 
+func (p *Plugin) playbooks() error {
+	var playbooks []string
+fs:
+	for _, p := range p.Config.Playbooks {
+		files, err := filepath.Glob(p)
+		// can there be a invalid glob pattern that still is a valid file name?
+		// just add it back to the list and let ansible return error out instead.
+		if err != nil {
+			playbooks = append(playbooks, p)
+			continue fs
```

Comment:
Just drop the fs part? Continue should get into the next loop run.

Reply:
Sure,. I prefer to use labels almost everywhere i break or continue to make it easier to read though and less error prone when making changes in larger structures.